### ISSUE TITLE
 Code: 1001, type: cppkafka::ConfigException "queued.min.messages" value 100000000

### ISSUE THREAD

**Issue by kernel288** [ClickHouse/ClickHouse#63110]
HI I upgrade my ch from 23.3.17 to version 23.8.8 revision 54465.

I look this 
https://github.com/ClickHouse/ClickHouse/issues/21648 but not work
my  kafak comsumer is error

```python
2024.04.29 19:42:52.781389 [ 19165 ] {403bd065-22e1-45e4-8a35-0b037b892c2e} <Trace> ContextAccess (default): Access granted: CREATE TABLE ON ztmdb.xxxx_kafka
2024.04.29 19:42:52.783354 [ 19165 ] {403bd065-22e1-45e4-8a35-0b037b892c2e} <Error> StorageKafka (xxxx_kafka): std::exception. Code: 1001, type: cppkafka::ConfigException, e.what() = Failed to set queued.min.messages: Configuration property "queued.min.messages" value 100000000 is outside allowed range 1..10000000

i set kafka in 

<!--
  NOTE: User and query level settings are set up in "users.xml" file.
  If you have accidentally specified user-level settings here, server won't start.
  You can either move the settings to the right place inside "users.xml" file
   or add <skip_check_for_incorrect_settings>1</skip_check_for_incorrect_settings> here.
-->
<clickhouse>
    <kafka>
     <queued_min_messages>10000000</queued_min_messages>
    </kafka>
    <logger>

```  

**But it's not work.  I don't know where to fix it ?  help. 
default min is big one , I need modify**

**Comment by den-crane** [comment]
Have you restarted Clickhouse after that?

**Comment by den-crane** [comment]
>"users.xml" file.

you need to add these settings to config.xml

**Comment by kernel288** [comment]
> Have you restarted Clickhouse after that?

I reboot the clickhouse .   
it's not work.

**Comment by kernel288** [comment]
> > "users.xml" file.
> 
> you need to add these settings to config.xml

**I add this file /etc/clickhouse-server/config.xml
<clickhouse>
....
    <kafka>
     <queued_min_messages>10000000</queued_min_messages>
    </kafka>
....
</clickhouse>**

**Comment by den-crane** [comment]
Check / share

```
show create table ztmdb.xxxx_kafka

**Comment by kernel288** [comment]
> Check / share
> 
> ```
> show create table ztmdb.xxxx_kafka
> ```

```python

CREATE TABLE ztmdb.xxxx_kafka
(
    `xxx` Nullable(DateTime),
    `bbb` String
)
ENGINE = Kafka
SETTINGS kafka_broker_list = 'wu01.kbb.com:9092,wu02.kbb.com:9092,wu03.kbb.com:9092', kafka_topic_list = 'xxxx_TOPIC', kafka_group_name = 'xxx_CONSUMER', kafka_format = 'JSONEachRow', kafka_row_delimiter = '\n'

``` 

This is my kafka table

**Comment by kernel288** [comment]
I check the create kafka table  not  queued_min_messages setting

CREATE TABLE [IF NOT EXISTS] [db.]table_name [ON CLUSTER cluster]
(
    name1 [type1] [ALIAS expr1],
    name2 [type2] [ALIAS expr2],
    ...
) ENGINE = Kafka()
SETTINGS
    kafka_broker_list = 'host:port',
    kafka_topic_list = 'topic1,topic2,...',
    kafka_group_name = 'group_name',
    kafka_format = 'data_format'[,]
    [kafka_schema = '',]
    [kafka_num_consumers = N,]
    [kafka_max_block_size = 0,]
    [kafka_skip_broken_messages = N,]
    [kafka_commit_every_batch = 0,]
    [kafka_client_id = '',]
    [kafka_poll_timeout_ms = 0,]
    [kafka_poll_max_batch_size = 0,]
    [kafka_flush_interval_ms = 0,]
    [kafka_thread_per_consumer = 0,]
    [kafka_handle_error_mode = 'default',]
    [kafka_commit_on_select = false,]
    [kafka_max_rows_per_message = 1];

**Comment by kernel288** [comment]
kafka setting 

<img width="1023" alt="image" src="https://github.com/ClickHouse/ClickHouse/assets/11566386/e3908bc3-1ea5-4d2a-a4c9-2b4ad603decf">

**Comment by den-crane** [comment]
@filimonov

**Comment by kernel288** [comment]
> @filimonov

this   is will set  queued_min_messages  .
            <max_insert_block_size>10000000</max_insert_block_size>

I set  max_insert_block_size  to 10000000 ..   the kafka it work .

**Comment by den-crane** [comment]
* https://github.com/ClickHouse/ClickHouse/pull/56121

**Comment by kernel288** [comment]
> fixed here:
> 
> * [Avoid problem with configuration queued.min.messages #56121](https://github.com/ClickHouse/ClickHouse/pull/56121) (23.10)
> 
> not backported to 23.8

OK  I will upgrade . I need time to test it

**Comment by kernel288** [comment]
The web site not 23.10 version 
https://packages.clickhouse.com/rpm/lts/

**Comment by den-crane** [comment]
23.10 is not LTS. LTS are .8 (August) and .3 (March).

23.8 - LTS, 24.3 - LTS

**Comment by kernel288** [comment]
thank you for reply 
24.3 - Did LTS fix the bug?

**Comment by den-crane** [comment]
24.3 includes this fix https://github.com/ClickHouse/ClickHouse/pull/56121

**Comment by kernel288** [comment]
> 24.3 includes this fix #56121

thank you
